gnosis/safe-react
# Add network routes to set network

## What it solves
Resolves #2800

## How this PR fixes it
Routes named the same as previous subdomains, i.e. `rinkeby`, `mainnet`, etc. have been added that accordingly set the `networkId` and redirect to the `ROOT_ROUTE`.

The last viewed safe logic is still existant and will then be handled by `<Route path={ROOT_ROUTE} ... />` as it has always been with `defaultSafe`.

## How to test it
Rinkeby is the default network in the development environment and Mainnet in staging/production. If you open `.../app/xdai`, you should see that the network is set to xDai. If you have a last viewed Safe on xDai, it will be open, otherwise the welcome page will be shown.
